todo-backend/src/entities/db.ts

```diff
@@ -0,0 +1,18 @@
+import { Options } from "sequelize";
+
+const {
+  DATABASE_NAME,
+  DATABASE_USER,
+  DATABASE_PASSWORD,
+  DATABASE_SVC,
+  DATABASE_PORT,
+} = process.env;
+
+export const dbConnectionOptions: Options = {
```

Comment:
these consts are potential candidates for configmaps and secrets via dotenv pkg?

Reply:
Yep, these should pick up the values from their corresponding environment variables.